CHANGELOG.md

```diff
@@ -42,6 +42,11 @@ Documentation:
 
 Compatibility:
 
+Unfortunately a number of tile sources are no longer publicly
```

Comment:
Great, but sadly need you to add this to doc/releases.rst as well.

Reply:
Thanks for the reminder. I'll do that now.

Reply:
Updated in 32d4af7bb